packages/cxl-ui/src/components/cxl-app-layout.js

```diff
@@ -111,6 +111,14 @@ export class CXLAppLayoutElement extends LitElement {
     `;
   }
 
+  updated(changedProperties) {
+    super.updated(changedProperties);
+
+    if (changedProperties.has('wide') && !this.wide) {
+      this.asideOpened = false;
```

Comment:
Do we need to still keep track of what the state was on `wide`, so it returns to such, when toggling between mobile / desktop.

Reply:
Updated to restore drawer state on media change